example/pubspec.lock

```diff
@@ -1,62 +1,34 @@
 # Generated by pub
 # See https://dart.dev/tools/pub/glossary#lockfile
 packages:
-  archive:
```

Comment:
We should not update .lock files all the time.

A lock file is exactly to avoid dependencies to update and to ensure the code will always work.

If we decide to change a lock file is either because we are adding new dependencies or we need to upgrade the versions to use a new feature. Both cases need a proper commit message to handle it in a separated commit.

Reply:
Noted.